draft-ietf-oauth-resource-metadata.xml

```diff
@@ -243,26 +241,10 @@
 	    JSON array containing a list of the JWS <xref target="JWS" /> signing algorithms
 	    (<spanx style="verb">alg</spanx> values) <xref target="JWA" />
 	    supported by the protected resource
-	    for signed content.
-	    The value <spanx style="verb">none</spanx> MAY be included.
-	  </t>
-
-	  <t hangText="resource_encryption_alg_values_supported">
-	    <vspace/>
-	    OPTIONAL.
-	    JSON array containing a list of the JWE <xref target="JWE" /> encryption algorithms
-	    (<spanx style="verb">alg</spanx> values) <xref target="JWA" />
-	    supported by the protected resource
-	    for encrypted content.
-	  </t>
-
-	  <t hangText="resource_encryption_enc_values_supported">
-	    <vspace/>
-	    OPTIONAL.
-	    JSON array containing a list of the JWE encryption algorithms
-	    (<spanx style="verb">enc</spanx> values) <xref target="JWA" />
-	    supported by the protected resource
-	    for encrypted content.
+	    for signing resource responses,
+	    as described in Section 5.6.2 of <xref target="FAPI.MessageSigning"/>.
```

Comment:
This reads like FAPI Message Signing is the only means of signing resource responses that this is applicable to, which I don't think is intended? Maybe qualify it with a "such as described"... 

Also FWIW there is no Section 5.6.2 in the referenced version of FAPI 2.0 Message Signing.  It's in the WG/editors copy but not the published draft. 

And (draft) FAPI Message Signing uses HTTP Message Signatures, which allows for or accommodates JWS Algorithms https://datatracker.ietf.org/doc/html/rfc9421#name-json-web-signature-jws-algo but is not limited to them https://datatracker.ietf.org/doc/html/rfc9421#section-3.3 / https://www.iana.org/assignments/http-message-signature/http-message-signature.xhtml#signature-algorithms so there's kind of an impedance mismatch in this parameter that is defined as `a list of the JWS <xref target="JWS" /> signing algorithms` and applicability to FAPI Message Signing and HTTP Message Signatures in general.